moc-health-check/README.md

```diff
@@ -0,0 +1,63 @@
+# MOC Health Reporter - v1.0
+
+********
+***Overview***
+********
+
+This repository provides a real-world application for pfurl project. It sends a generic command from pfurl to send local files to Pfioh, run a sample job on pman, and fetch the files back from pfioh. Overall, it tests all of the core elements of Chris Platform with the requirement of only one job id, making the process reentrant. 
+
+
+- ``MOC Status Reporter``: a program to test the functionality of MOC 
+
+MOC Health Reporter
+=====
+
+``MOC Health Reporter`` is a plugin for pfurl utility, since it successfully utilizes pfurl as a testing service for ChRIS Research Integration System.
+
+In simple terms, ``MOC Health Reporter`` is a status fetcher used to send http-based messages to remote services such as ``pman`` and ``pfioh``, in order to test their reliability and response time. 

```

Comment:
Let's just explain it once rather than once and then in simpler terms.  Pick one good version and go with it.